examples/react/src/app.jsx

```diff
@@ -1,21 +0,0 @@
-import * as React from 'react';
```

Comment:
The idea of app.js was to show an app which is entirely independent from gondel (because it uses just the button wrapper)

Could we please keep it simple here?